addons/block_code/ui/blocks/utilities/template_editor/template_editor.gd

```diff
@@ -137,5 +141,12 @@ func _append_output_parameter(parameter: Dictionary, id: int):
 	_container.add_child(parameter_output)
 
 
+func _append_const_parameter(parameter: Dictionary, id: int):
+	# const_parameter is a kind of in_parameter with default value, but never
+	# changes value.
+	var parameter_const := _append_input_parameter(parameter, id)
+	parameter_const.visible = false
```

Comment:
What does this do exactly? I want to make sure the extra syntax is really worth it since it seems like the regular input parameter would work fine.

Reply:
This clears the in parameter's "visible" to avoid modify the value from users.

Reply:
Also, if the in parameter shows up, it displays the full file path as a long text.

Reply:
* Without the input parameter in the display:
  ![Screenshot From 2024-11-27 17-10-07](https://github.com/user-attachments/assets/1859547d-d880-4064-bc87-b591dd294514)
* With the input parameter in the display:
  ![Screenshot From 2024-11-27 17-12-16](https://github.com/user-attachments/assets/4367a76c-5211-49d8-8dc0-4aeda0cc004b)